conda-forge.yml

```diff
@@ -13,4 +13,3 @@ provider:
   linux: azure
   osx: azure
   win: azure
-upload_on_branch: master
```

Comment:
@jklynch, I don't think we should remove it. Could you please revert it?

Reply:
reverted

Reply:
Great, thanks. Was the feedstock rerendered? It should be done to propagate the changed parameters from `conda-forge.yml` to `.azure-pipelines/...` config files.